lib/Doctrine/ORM/Tools/Pagination/CountOutputWalker.php

```diff
@@ -71,9 +71,7 @@ public function __construct($query, $parserResult, array $queryComponents)
      */
     public function walkSelectStatement(SelectStatement $AST)
     {
-        if ($this->platform instanceof SQLServerPlatform) {
```

Comment:
Now that `$this->platform` is not used anymore, we need to remove the property from the class and the assignment from the constructor, to make PHPStan and Psalm happy.